lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -144,6 +144,16 @@ describe('CategoryRenderer', () => {
 
     const audits = categoryDOM.querySelectorAll('.lh-audit');
     assert.equal(audits.length, category.auditRefs.length, 'renders correct number of audits');
+
+    assert.ok(categoryDOM.querySelector('.lh-gauge__wrapper--plugin') === null);
+  });
+
+  it('plugin category has plugin badge', () => {
+    const category = sampleResults.reportCategories.find(c => c.id === 'seo');
+    // Any category not of the core
+    category.id = 'someplugin';
```

Comment:
this taints the sampleResults in all remaining tests. you should clone them with the JSON.stringify/parse dance first.

Reply:
done